src/tests/suite/daos_container.c

```diff
@@ -2602,10 +2603,15 @@ co_redun_lvl(void **state)
 	/* test 1 - cont rf and obj redundancy */
 	print_message("verify cont rf is set and can be queried ...\n");
 	if (arg->myrank == 0) {
-		rc = daos_cont_query(arg->coh, &info, NULL, NULL);
+		prop_out = daos_prop_alloc(2);
```

Comment:
this could fail?

Reply:
Ah yes. I see that most of the daos_test calls to daos_prop_alloc() are followed by an assert_non_null() call so I can add one here.